### ISSUE TITLE
functions in current module dont autocomplete if after ->

### ISSUE THREAD

**Issue by kswope** [rescript-lang/rescript-vscode#901]
this autocompletes "mylog"

```
let mylog = (s: string) => Console.log(s)

mylog("hello")
```

this just prematurely offers a bunch of String module functions even before I begin typing mylong but never completes mylog as Im typing it

```
let mylog = (s: string) => Console.log(s)

"hello"->mylog

```

Seems to be the same for "opened" modules and even ones opened with bsc-flags from rescript.json

**Comment by zth** [comment]
Yeah this is a matter of pipe completions being specialized and not including regular results. Should be a fairly simple fix.
